manager/controllers/app/m4dapplication_controller_test.go

```diff
@@ -228,22 +228,22 @@ var _ = Describe("M4DApplication Controller", func() {
 			// Create M4DApplication
 			Expect(k8sClient.Create(context.Background(), resource)).Should(Succeed())
 
-			By("Expecting an error condition with reason Denied")
-			Eventually(func() []apiv1alpha1.Condition {
+			By("Expecting access denied on read")
+			Eventually(func() string {
 				f := &apiv1alpha1.M4DApplication{}
 				_ = k8sClient.Get(context.Background(), appSignature, f)
-				return f.Status.Conditions
+				return f.Status.Error
 			}, timeout, interval).ShouldNot(BeEmpty())
 
 			_ = k8sClient.Get(context.Background(), appSignature, resource)
-			Expect(resource.Status.Conditions[0].Reason).To(Equal("AccessDenied"))
+			Expect(resource.Status.Error).To(ContainSubstring("Governance policies forbid access to the data"))
```

Comment:
Too easy to break. Use consts for the error messages.

Reply:
OK.